encrypted-media/scripts/playback-temporary-events.js

```diff
@@ -76,15 +76,15 @@ function runTest(config,qualifier) {
         }
 
         function onClosed(event) {
-            _events.push('closed-promise');
+            _events.push('closed-method-promise-resolved');
```

Comment:
Yes, Chrome does not currently implement the correct order. I filed https://crbug.com/659374. I have https://codereview.chromium.org/2446993003/ to fix Clear Key, but fixing Widevine will take more time.